clients/web/src/views/layout/FooterView.js

```diff
@@ -9,9 +9,18 @@ import 'girder/stylesheets/layout/footer.styl';
  * This view shows the footer in the layout.
  */
 var LayoutFooterView = View.extend({
+
+    initialize: function (settings) {
+        const mail = settings.contactEmail || '';
+        if (mail !== '') {
+            this.emailHref = 'mailto:' + mail;
```

Comment:
Use [an ES6 template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for this concatenation.